src/SFA.DAS.ApplyService.Web/ViewModels/Roatp/TaskListViewModel.cs

```diff
@@ -64,15 +215,15 @@ public bool IntroductionPageNextSectionUnavailable(int sequenceId, int sectionId
                 foreach(var section in yourOrganisationSequence.Sections)
                 {
                     var sectionStatus = RoatpTaskListWorkflowService.SectionStatus(ApplicationSequences, NotRequiredOverrides, RoatpWorkflowSequenceIds.YourOrganisation, section.SectionId, ApplicationRouteId);
-                    if (sectionStatus.ToLower() != PageStatusCompleted)
+                    if (sectionStatus != TaskListSectionStatus.Completed)
```

Comment:
Sorry, just seen this. To make it safer I would do  if (!TaskListSectionStatus.Completed.Equals(sectionStatus, StringComparison.InvariantCultureIgnoreCase))

Reply:
The SectionStatus function now only returns one of the TaskListSectionStatus constant values, so case checking not required here